.travis.yml

```diff
@@ -31,6 +34,9 @@ jobs:
     - stage: test
       script:
       - |
+        # test to see if decryption works
```

Comment:
I'm glad you got it working! Still not quite sure what I did wrong the first time but hopefully I won't have to do it again. 😀